lib/src/stream_sink_transformer/handler_transformer.dart

```diff
@@ -88,8 +88,8 @@ class _HandlerSink<S, T> implements StreamSink<S> {
             // stack trace arg non-nullable.
```

Comment:
nit: remove this todo

Reply:
actually now that I have read it lol we should just pass a tearoff of `_transformer._handleError` if we can now